### PR TITLE
added wizard section

### PR DESCRIPTION
**Pull request by Pratichhya**

### CODE REVIEW

**Comment by Pratichhya** [comment]
@soxofaan @jdries Do you suggest this section might require more detailed information or to be limited with this for the moment

File: Applications/WebEditor.qmd [hunk 1]
```diff
@@ -96,3 +96,16 @@ Once the execution process is completed, the result is automatically saved local
 Furthermore, if you have created Batch Job, you can monitor its action from the same window. 
 
 
+## Data Download using Wizard
+
+The Wizard feature in the top navigation bar allows easy creation of processes for common usecases without having to drag and drop the available funtions as done earlier.
```
Comment (soxofaan) [inline Applications/WebEditor.qmd:101]:
```suggestion
The Wizard feature in the top navigation bar allows easy creation of process graphs for common, basic use cases without having to drag and drop processes as done earlier. 
```

File: Applications/WebEditor.qmd [hunk 2]
```diff
@@ -96,3 +96,16 @@ Once the execution process is completed, the result is automatically saved local
 Furthermore, if you have created Batch Job, you can monitor its action from the same window. 
 
 
+## Data Download using Wizard
+
+The Wizard feature in the top navigation bar allows easy creation of processes for common usecases without having to drag and drop the available funtions as done earlier. 
+
+:::{callout-note}
+
+This feature is experimental and there are chances that the generated models won't work.
```
Comment (soxofaan) [inline Applications/WebEditor.qmd:105]:
```suggestion
This feature is experimental and it's not guaranteed that the generated process graphs are fully functional.
```

File: Applications/_images/gif/widzard.gif
Comment (soxofaan) [inline Applications/_images/gif/widzard.gif:1]: why is this file added if it is not used?
Also note the typo in the file name

File: Applications/WebEditor.qmd (hunk 1 above)
Comment (Pratichhya) [reply Applications/WebEditor.qmd:101]: updated

File: Applications/WebEditor.qmd [hunk 3]
```diff
@@ -96,3 +96,16 @@ Once the execution process is completed, the result is automatically saved local
 Furthermore, if you have created Batch Job, you can monitor its action from the same window. 
 
 
+## Data Download using Wizard
+
+The Wizard feature in the top navigation bar allows easy creation of processes for common usecases without having to drag and drop the available funtions as done earlier. 
+
+:::{callout-note}
```
[the remark this answers is not in the document]
Comment (Pratichhya) [reply Applications/WebEditor.qmd:103]: changed note to warning

File: Applications/WebEditor.qmd (hunk 2 above)
Comment (Pratichhya) [reply Applications/WebEditor.qmd:105]: updated to suggested sentence

File: Applications/_images/gif/widzard.gif
Comment (Pratichhya) [reply Applications/_images/gif/widzard.gif:1]: Removed the unnessaary file

**Comment by Pratichhya** [comment]
@soxofaan Sorry for the confusion. When trying to keep updated PR, it seems like I messed it up with a long list of changes. Thus created a new PR based on this( #233 ). It adopts all the changes that you suggested.
